samples/wifi/shell/overlay-tfm.conf

```diff
@@ -26,4 +26,4 @@ CONFIG_NET_RX_STACK_SIZE=2048
 CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
 
 CONFIG_MBEDTLS_PSA_CRYPTO_C=y
-CONFIG_MBEDTLS_HEAP_SIZE=1024
+CONFIG_MBEDTLS_HEAP_SIZE=16384
```

Comment:
For a basic TF-M without Wi-Fi credentials, we don't need this change right? So, may be just rely on defaults?

Reply:
If a user applies the overlay-tfm.conf overlay my patch won't work due to the overlay kconfigs taking precedence.
How about moving all of these to the the boards/_ns files and keeping the heap size.

Then you don't have do include this overlay when building with non-secure.

Reply:
Yes, that would be better.

Reply:
fixed, please see the changes.